app/code/core/Mage/CatalogInventory/Model/Observer.php

```diff
@@ -1041,10 +1041,21 @@ public function refundOrderItem($observer)
      */
```

Comment:
```suggestion
     * @return void
     */
```

just to trigger the workflow, they do not run on a force push (for rebase)